docs/tutorials/elasticdl_local.md

```diff
@@ -56,14 +56,23 @@ The following command to start a Kubernetes cluster locally using Minikube.
 minikube start --vm-driver=hyperkit \
   --cpus 2 --memory 6144 --disk-size=50gb \
   --mount=true --mount-string="./data:/data"
-kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
 eval $(minikube docker-env)
 ```
 
 The above command-line option `--mount-string` exposes the directory `./data` on
 the host to Minikube, so we can bind mount it into containers running on the
 local Kubernetes cluster.
 
+The following command is necessary to enable [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+of Kubernetes. At first, we need retrieves the source code to get the RBAC
```

Comment:
ElasticDL needs permissions to access Kubernetes objects. Permissions are authorized by  [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/). The RBAC configuration YAML file is in ElasticDL repository. Thus, we need to retrieve the file from ElasticDL repository before applying RBAC.